### PR TITLE
separate scheduler metrics from job_store metrics

### PR DESCRIPTION
**Pull request by h00dy**

### CODE REVIEW

File: apps/ui/lib/ui_web/prometheus.ex [hunk 1]
```diff
@@ -21,13 +21,15 @@ defmodule SchedulerInstrumenter do
...
-    Counter.declare([name: :scheduler_processed_jobs_total,
+    Counter.declare([name: :scheduler_running_jobs_total,
                      help: "Scheduled jobs count.",
                      labels: [:env]])
     Gauge.declare([name: :db_jobs_total,
                   help: "Nr of jobs in JobStore.",
                   labels: [:env]])
-
+    Gauge.declare([name: :scheduler_jobs_total,
+                  help: "Nr of jobs in JobStore.",
```
Comment (pnuckowski) [inline apps/ui/lib/ui_web/prometheus.ex]: same `help` description as for `: db_jobs_total`?

File: apps/scheduler/lib/scheduler.ex [hunk 2]
```diff
@@ -5,7 +5,7 @@ defmodule Scheduler do
   alias Scheduler.Dispatcher
   require Logger
   use GenServer
-  use Prometheus.Metric
+  # use Prometheus.Metric
```
Comment (pnuckowski) [inline apps/scheduler/lib/scheduler.ex]: Is no longer used?

File: apps/ui/lib/ui_web/prometheus.ex [hunk 3]
```diff
@@ -37,18 +39,51 @@ defmodule SchedulerInstrumenter do
...
+  ## Parameters
+  - env: check environment
+  """
+  def inc_running_jobs(env) do
+    Counter.inc([name: :scheduler_running_jobs_total,
+                labels: [env]])
+  end
+
+  @doc """
+  Increments counter measuring jobs added to scheuduler
```
Comment (pnuckowski) [inline apps/ui/lib/ui_web/prometheus.ex]: typo

File: apps/ui/lib/ui_web/prometheus.ex (hunk 1 above)
Comment (h00dy) [reply apps/ui/lib/ui_web/prometheus.ex]: good catch :)

File: apps/scheduler/lib/scheduler.ex (hunk 2 above)
Comment (h00dy) [reply apps/scheduler/lib/scheduler.ex]: Yes, I'll remove this line, to keep it clean.

File: apps/ui/lib/ui_web/prometheus.ex (hunk 3 above)
Comment (h00dy) [reply apps/ui/lib/ui_web/prometheus.ex]: fixed

**Pull request merged**